claraabk/projetoIP
# (hotfix):changed background and main sprite

## What I did

- Changed background to definitive background
- Changed the hero sprite to monica sprite


## How to test

1. Open the terminal and go to the folder of this project

1. In terminal:
   1. `git checkout develop`
   1. `git pull`
   1. `git checkout  hotfix/back_and_sprite`
   1. `git pull origin hotfix/back_and_sprite`
  
1. In VScode open `background.py and monica.py`

1. Things to do during the code:

   - Open insomnia and postbird to test the requisitions needed

1. Read the changed files section on GitHub

1. Test if monica is flipping and if the background is ok 

&nbsp;

---

**Time spent:** <see your time spent on https://wakatime.com/dashboard >
